docs/components/password/test.html

```diff
@@ -0,0 +1,385 @@
+            </td>
+            <td>
+              {{passwordInput('invalid', className="hxInvalid", isValid=false, isDisabled=true)}}
+              {{passwordInput('invalid', className="hxInvalid", isValid=false, isDisabled=true, isFocused=true)}}
+            </td>
+          </tr>
+        </tbody>
+      </table>
+    </section>
+
+    <section>
+      <header>
+        <h3>Required</h3>
+        <code>&lt;input type="password" required&gt;</code>
+      </header>
+      <table>
+        <thead>
+          <tr>
+            <th>:valid</th>
+            <th>:invalid</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>
+              {{passwordInput('invalid', className="hxInvalid", isRequired=true)}}
+              {{passwordInput('invalid', className="hxInvalid", isRequired=true, isFocused=true)}}
+            </td>
+            <td>
+              {{passwordInput('invalid', className="hxInvalid", isRequired=true, isValid=false)}}
+              {{passwordInput('invalid', className="hxInvalid", isRequired=true, isValid=false, isFocused=true)}}
+            </td>
+          </tr>
+          <tr>
+            <td>
+              {{passwordInput('invalid', className="hxInvalid", isRequired=true, isDisabled=true)}}
+              {{passwordInput('invalid', className="hxInvalid", isRequired=true, isDisabled=true, isFocused=true)}}
+            </td>
+            <td>
+              {{passwordInput('invalid', className="hxInvalid", isRequired=true, isValid=false, isDisabled=true)}}
+              {{passwordInput('invalid', className="hxInvalid", isRequired=true, isValid=false, isDisabled=true, isFocused=true)}}
+            </td>
+          </tr>
+        </tbody>
+      </table>
+    </section>
+  </section>
+
+  <section>
+    <header>
+      <h2 id="test-with-placeholder">With Placeholder</h2>
+      <code>&lt;input type="password" placeholder="Please Enter Password" /&gt;</code>
+    </header>
+
+    <div class="hxRow">
+      <div class="hxCol hxSpan-4">
+        <hx-password-control>
+          <input
+            placeholder="Please Enter Password"
+            type="password"
+          />
+          <label>Valid w/ Placeholder</label>
+        </hx-password-control>
+      </div>
+      <div class="hxCol hxSpan-4">
+        <hx-password-control class="hxInvalid">
+          <input
+            placeholder="Please Enter Password"
+            type="password"
+          />
+          <label>Invalid w/ Placeholder</label>
+        </hx-password-control>
+      </div>
+      <div class="hxCol hxSpan-4">
+        <hx-password-control>
+          <input
+            disabled
+            placeholder="Please Enter Password"
+            type="password"
+          />
+          <label>Disabled w/ Placeholder</label>
+        </hx-password-control>
+      </div>
+    </div>
+  </section>
+
+  <section>
+    <header>
+      <h2 id="test-layout">Layout</h2>
```

Comment:
### `Optional` tests can be removed:

<img width="799" alt="Screen Shot 2020-02-26 at 1 35 49 PM" src="https://user-images.githubusercontent.com/10120600/75380929-7ee0e880-589d-11ea-9c51-57e3d58f170d.png">

Reply:
Changes have been done.